### DISCUSSION TITLE
Number of Cards in Layout is maxed at 5

### DISCUSSION THREAD

**Discussion by sanjeevba** [microsoft-search/pnp-modern-search#4200] [General]
Version used
**Ex: 4.93**

I'm using the Cards layout in _Search Results_ and I've set the 'Preferred number of cards per row' to 6. I have a large monitor but I am unable to see more that 5 cards in a row. 

How can I see more cards on a given row. 

If necessary, I am willing to switch to a Custom layout and add the necessary styling. I tried a few stylings but none of them seemed to work

**Comment by jblueml** [comment]
@sanjeevba 
The horizontal space available for the Search Results webpart is generally limited to the maximum width of the page-section the webpart is placed in.

And due to the fact, that the minimum width of a document-card internally is 206 pixels + some pixels for padding and the maximum width of a 1-column section is about 1236 pixels, no more than 5 cards can be displayed side-by-side.

An option would be to simply place the Search Results webpart in a full-width-section and edit the Handlebars-template in the following way:
![image](https://github.com/user-attachments/assets/787673d8-e678-4040-bd6f-bddd08830bd8)
Of course you can set a different percentage-value to have less or more cards side-by-side.

_Note: this isn't a nice approach - but it works 😉_
